src/ape/api/transactions.py

```diff
@@ -294,6 +294,18 @@ def validate_txn_hash(cls, value):
     def call_tree(self) -> Optional[Any]:
         return None
 
+    @cached_property
+    def debug_logs(self) -> List[List[Any]]:
+        """Return any debug log data outputted by the transaction"""
+        return []
+
+    def print_debug_logs(self):
+        """
+        Log debug logs to the console
+        """
+        for ln in self.debug_logs:
+            logger.info("[CONTRACT-LOG] ".join(map(str, ln)))
```

Comment:
worry: `CONTRACT-LOG` seems like an overloaded term which may be confusing. It makes me think it is a "contract log" like an event.

Reply:
I just took the recommendation.  What do you think would be better?

Reply:
`DEBUG-LOG` I think is better, I saw that somewhere else and I knew it was different than `.logs`.